docs/self-managed/platform-deployment/helm-kubernetes/deploy.md

```diff
@@ -244,31 +252,41 @@ postgresql:
   enabled: false
 ```
 
-#### Install the Helm chart
+For more details, check [Web Modeler Helm values](https://artifacthub.io/packages/helm/camunda/camunda-platform#webmodeler-parameters).
 
-Assuming you have saved your configuration in `modeler-values.yaml`, install the Helm chart by running the following:
+### Install Console
```

Comment:
When I follow these instructions, I can not access the console. I need to figure out connectivity and this doc does not help me there.